test-apps/jira-damn-vulnerable-forge-app/src/utils.js

```diff
@@ -1,21 +1,119 @@
 import api, { route } from '@forge/api';
+import jwt from 'jsonwebtoken';
+import * as atlassian_jwt from 'atlassian-jwt';
+import * as cryptoJS from 'crypto-js';
+import * as jwtSimple from 'jwt-simple';
+import { testFunctionFromTestFile } from './testfile';
+import module_exports_func from './moduleex.js';
+import { func_from_exports, diffunc } from './exportse.js';
+import { another_export as foo, newExport } from './newexports.js';
+import func_defult from './export_default';
+import my_function from './export_default2.js';
+import { classone } from './anewclass';
+import * as c1 from './anewclass';
+/*
+import {default as something} from what;
+import something from what;
 
-export async function fetchIssueSummary(issueIdOrKey) {
-  /* const api = await import('@forge/api'); */
-  const resp = await api
-    .asApp()
-    .requestJira(route`/rest/api/3/issue/${issueIdOrKey}?fields=summary`, {
-      headers: {
-        Accept: 'application/json',
-      },
-    });
+packagename: jsonwebtoken
+identifier: default
+type: Object("sign")
+position: 4
+
+function require(input) {}
+require('foo');
+
+type: Enum {
+  Function,
+  Object(String), // <- method_name
+}
+*/
+
+// import HmacMD5 from 'crypto-js';
+// DefId(4)
+
+// DEFINITIONS
+// 0 -> name
+// 0 -> Definition(enum)
+
+// function chair() {
+// DefId(22)
+//as_foreign_import(DefId(22), 'newexports')) -> Some(ImportKind("another_export"))
+// foo();
+//
+// [DefId(4), Static('blah'), Static('open')]
+// atlassian_jwt['blah' + 'test']();
+// atlassian_jwt.blah.open();
+// atlassian_jwt['blah']();
+
+// const blah = 'sign';
+// atlassian_jwt[blah]();
+// atlassian_jwt.sign();
+// }
+
+let global = 'test';
+
+export async function fetchIssueSummary(issueIdOrKey, test_value) {
+  let obj = {
+    method: 'POST',
+    bananas: 'apple',
+    headers: {
+      //
+      Accept: 'application/json',
+    },
+  };
+
+  module_exports_func();
+  func_from_exports();
+  another_export();
+  newExport();
+  func_defult();
+  my_function();
+  let global = 'test';
+
+  // testing all the libraries
+  // var token = jwt.sign({ foo: 'bar' }, 'peek a boo');
+  // var hmac = HmacMD5('Secret Message', 'HMAC PASSWORD');
```

Comment:
Can you add test cases for named imports as well? Also for cases where an identifier could shadow an import to see if we don't raise a false positive there.